recipes/librdkafka/all/test_package/test_package.cpp

```diff
@@ -3,9 +3,10 @@
 
 int main(int argc, char const *argv[]) {
   rd_kafka_conf_t *conf = rd_kafka_conf_new();
+
   std::cout << std::endl
             << "----------------->Tests are done.<---------------------" << std::endl
-	    << "Using version " << rd_kafka_version_str() << std::endl
+            << "Using version (from C lib) " << rd_kafka_version_str() << std::endl
             << "///////////////////////////////////////////////////////" << std::endl;
```

Comment:
I would replace iostream by stdio.h (to use printf) and change extension from .cpp to .c